service/controller/v5/version_bundle.go

```diff
@@ -8,13 +8,18 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "azure-operator",
-				Description: "Add your changes here.",
-				Kind:        versionbundle.KindAdded,
+				Component:   "cloudconfig",
+				Description: "Updated Calico to 3.2.3.",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "cloudconfig",
+				Description: "Updated Calico manifest with resource limits.",
+				Kind:        versionbundle.KindChanged,
 			},
```

Comment:
No need to change calico in azure-specific manifests?

Reply:
done here https://github.com/giantswarm/azure-operator/pull/378